src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -43,6 +48,15 @@ private protected DataViewType(Type rawType)
     /// <summary>
     /// The abstract base class for all non-primitive types.
     /// </summary>
+    /// <remarks>
+    /// This class stands in constrast to <see cref="PrimitiveDataViewType"/>. As that class is defined
+    /// to encapsulate cases where instances of the reference type can be freely copied without concerns
```

Comment:
`instances of the reference type`

Did you mean

`instances of the representation type`?